.buildkite/scripts/steps/functional/apm_cypress.sh

```diff
@@ -12,8 +12,11 @@ APM_CYPRESS_RECORD_KEY="$(retry 5 5 vault read -field=CYPRESS_RECORD_KEY secret/
 export JOB=kibana-apm-cypress
 IS_FLAKY_TEST_RUNNER=${CLI_COUNT:-0}
 
-# Disable parallel tests and dashboard recording when running them in the flaky test runner
-if [[ "$IS_FLAKY_TEST_RUNNER" -ne 1 ]]; then
+# Disable parallel tests and dashboard recording when
+# running them in the flaky test runner
+# PR is not labeled with apm:cypress-record
+
```

Comment:
Maybe it's just me but I had a hard time reading through this
Could we adjust this comment here?
What do you think about:
```
Enabling cypress dashboard recording when
PR is labeled with `apm:cypress-record` and
we are not using the flaky test runner
```
?

Reply:
@yngrdyn 👍 how about in one line? I think it's more readable